tests/test_midi/src/app_midi_simple.xc

```diff
@@ -0,0 +1,186 @@
+// Copyright 2024 XMOS LIMITED.
+// This Software is subject to the terms of the XMOS Public Licence: Version 1.
+
+/* A very simple *example* of a USB audio application (and as such is un-verified for production)
+ *
+ * It uses the main blocks from the lib_xua
+ *
+ * - 2 channels out I2S only
```

Comment:
This comments should be deleted (copy pasta)

Reply:
Ah makes more sense!